vmdb/app/controllers/report_controller/saved_reports.rb

```diff
@@ -24,8 +24,7 @@ def fetch_saved_report(id)
       return
     end
     @right_cell_text ||= _("%{model} \"%{name}\"") % {:name=>"#{rr.name} - #{format_timezone(rr.created_on,Time.zone,"gt")}", :model=>"Saved Report"}
-    user = User.find_by_userid(session[:userid])
-    if user.admin_user? || rr.miq_group_id == session[:group]
+    if current_user.admin_user? || rr.miq_group_id == session[:group]
```

Comment:
yes, you are right.

In this PR, I tried to limit to only 1 use case: `User.find_by_userid(session[:userid])`

In other PRs, I would like to:

1. introduce `login_as` in specs vs our current inconsistent way.
1. figure out how to introduce these changes into `ApplicationHelper`
1. replace `session[:userid]`: 206 across 47 files
1. replace `session[:username]`: 8 across 5 files
1. replace `session[:group]`: 27 matches in 7 files
1. replace `session[:role]`: huh, it isn't used?
1. replace `session[:userrole]`: 23 across 10 files - mostly just replacing with `current_user.super_admin?`